cylc/flow/network/server.py

```diff
@@ -858,12 +921,12 @@ def put_broadcast(
 
     @authorise(Priv.CONTROL)
     @expose
-    def put_ext_trigger(self, event_message, event_id):
+    def put_ext_trigger(self, message, id):
```

Comment:
`id` is also a built-in, but I think it's harmless here.

Reply:
So the function signature is bound to the mutation interface. The mutation has an argument called `id` so the method must have one too.

We can use a  `**kwargs` hack here but as you say it's harmless.

Reply:
Codacy has flagged this as an error, "redefining a built-in".  I suppose we should really change the mutation interface as well. (Can be a follow-up though).